pages/build-scripts.md

```diff
@@ -0,0 +1,81 @@
+
+```
+source <(curl -sSL "${SCRIPT_URL}/prepare-unstable.sh")
+```
+
+The *SCRIPT_DIR* variable should be configured to specify where the build scripts should be copied to, which should be set on the build stage containing the prepare script. For eaxmple, `./scripts/`.
+
+Similarly, the *SCRIPT_URL* environment variable should be set to the source location of the scripts you want to use. For example,
+
+```
+https://raw.githubusercontent.com/<github_user>/build-lib/master/src
+```
+
+where *github_user* would be `IBM-Blockchain-Starter-Kit` for the starter kit organisation, or your own GitHub user ID if you have forked the repository.
+
+## Using customised scripts
+
+In some cases you may wish to customise the build scripts to handle your own specific circumstances. In this case you can include modified copies of the build scripts in the source repository you are building.
+
+None of your own build scripts that exist in the *SCRIPT_DIR* directory in your source code repository will be overwritten when scripts are fetched in the prepare script, so you can override some or all of the build script library as required. Check the environment variable settings on each build stage to find the value of the *SCRIPT_DIR* variable.
+
+## Calling the build scripts
+
+If you are not using one of the starter kit toolchain templates, you need to add the following to your build stage jobs:
+
+```
+#!/bin/bash
+source "${SCRIPT_DIR}/router.sh" <stage> <platform>
+```
+
+where *stage* is the build stage (`build`, `test` or `deploy`) and *platform* is one of the supported platforms, described below.
+
+The *SCRIPT_DIR* variable should be configured in the build stage enviornment variables, and should match the same variable in the *PREPARE* stage where the scripts are downloaded.
+
+The router script will use the *stage* and *platform* arguments to run the appropriate scripts.
+
+### go-chaincode
+
+Use the `go-chaincode` platform to build Go chaincode.
+
+If you are starting a new chaincode project using the Go language, you can clone and modify the [chaincode-bootstrap](https://github.com/IBM-Blockchain-Starter-Kit/chaincode-bootstrap) project. Alternatively, you can generate a new Go chaincode project using the [Yeoman generator for Hyperledger Fabric](https://www.npmjs.com/package/generator-fabric).
+
+### js-chaincode
+
+Use the `js-chaincode` platform to build Node.js or TypeScript chaincode.
+
+If you are starting a new chaincode project using Node.js, you can clone and modify the [nodejs-chaincode-bootstrap](https://github.com/IBM-Blockchain-Starter-Kit/nodejs-chaincode-bootstrap) project. Alternatively, you can generate a new Go chaincode project using the [Yeoman generator for Hyperledger Fabric](https://www.npmjs.com/package/generator-fabric).
```

Comment:
>Alternatively, you can generate a new Go chaincode project...

Should this read Node.js/Typescript project?

Reply:
Yes it should!